src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,340 @@ textarea {
 
 }
 
+.input-group {
```

Comment:
BTWs, this is more of a just asking to see if it makes sense to remove. It's fine to leave in.